python/ray/serve/api.py

```diff
@@ -288,6 +289,7 @@ def deployment(
     graceful_shutdown_timeout_s: Optional[float] = None,
     health_check_period_s: Optional[float] = None,
     health_check_timeout_s: Optional[float] = None,
+    driver_deployment: Optional[bool] = False,
```

Comment:
since this is boolean, it should be called `is_driver_deployment`. but also user might be confused by this. how about `daemon_deployment` or `everynode`, or something..